src/elm/App/Update.elm

```diff
@@ -21,6 +21,10 @@ type Action
   -- NoOp actions
   | NoOp
 
+dragActions : Signal Action
+dragActions =
+    Signal.map ChildTimelineAction Timeline.Update.startTimeActions
```

Comment:
nice! so if we had multiple components, how would you delegate the drag action to all of them?